### ISSUE TITLE
[v1.2.0 BUG] Module will not found when I use img with public path

### ISSUE THREAD

**Issue by zprial** [doczjs/docz#863]
## Bug Report

**It will not working when I update to v1.2.0**

First, it's ok when I use v1.1.0, but, when I update to v1.2.0, it can't find public files like this:
![image](https://user-images.githubusercontent.com/33095380/57502283-a28e6e80-731d-11e9-96ce-8760dadf84cc.png)

then, when I comment it, there will be:
![image](https://user-images.githubusercontent.com/33095380/57502306-bb971f80-731d-11e9-81a4-b358c53b5882.png)

Here is my doczrc.js:
```js
const MiniCssExtractPlugin = require('mini-css-extract-plugin');
const path = require('path');
const webpackMerge = require('webpack-merge');
const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');

const resolve = (...paths) => path.resolve(__dirname, ...paths);

export default {
  theme: 'my-custom-theme',
  base: '/docs',
  ignore:['license.md'],
  src: './packages',
  public: '/public',
  modifyBundlerConfig: (config, dev) => {
    const styleLoaders = ['css-loader', 'postcss-loader'];
    if (dev) {
      styleLoaders.splice(0, 0, 'style-loader')
    } else {
      styleLoaders.splice(0, 0, {
        loader: MiniCssExtractPlugin.loader,
      })
    }
    return webpackMerge(config, {
      module: {
        rules: [
          {
            test: /\.css$/,
            use: styleLoaders
          },
          { test: /\.scss$/,
            use: [...styleLoaders, 'sass-loader']
          }
        ]
      },
      optimization: {
        minimizer: [
          new OptimizeCSSAssetsPlugin({})
        ]
      },
      plugins: [
        dev ? () => {} : new MiniCssExtractPlugin({
          filename: '[name].css',
        }),
      ]
    });
  }
}

```

**Expected behavior**

Is there has any way to solve it? thks

**Environment**

* OS: [OSX 10.14]
* Node/npm version: [Node v10.13.0 /npm v6.4.1]

**Comment by adrianschmidt** [comment]
I'm also having problems with not being able to load any files from the public-folder in 1.2.0.

Based on the above, I'll try downgrading to 1.1.0 and see if I can get it to work then.

**Comment by adrianschmidt** [comment]
@zprial BTW, I think you might have better luck getting a reply from someone working on the project if you change the title of this issue to something more descriptive. I think the issue body is pretty detailed, but when the title is basically" it doesn't work", I think most people will just ignore it, as that's the most hated "bug report" ever…

**Comment by zprial** [comment]
@adrianschmidt Thanks for your remind, I changed the title. And I found the reason for this error. When I insert an image, [remark-docz](https://github.com/pedronauck/docz/blob/master/core/remark-docz/src/index.ts#L72-L86) will replace my image's src.

![image](https://user-images.githubusercontent.com/33095380/59449041-e868b600-8e38-11e9-9348-06c0c45f6af9.png)

I dont't know why the author wants to write like this.

https://github.com/pedronauck/docz/blob/master/core/remark-docz/src/index.ts#L72-L86

**Comment by adrianschmidt** [comment]
Hmm… that path looks a bit wonky? Shouldn't it be `../public` rather than `".//public`?

**Comment by ejuo** [comment]
Using apostrophe like `public: '/public',` working for me.

**Comment by zprial** [comment]
Yes, I had already done that. But, it still not work for me, and I create a demo:

https://codesandbox.io/s/docz-bug-assets-cs38p

**Comment by ejuo** [comment]
You need specify relative patch where your mdx file located.
https://codesandbox.io/s/docz-bug-assets-ell2q

**Comment by zprial** [comment]
> @zprial
> You need specify relative patch where your mdx file located.
> https://codesandbox.io/s/docz-bug-assets-ell2q

🤔It's OK, but I think the `public` option should be used for static files rather than as a webpack module.

Like [docz.site](https://www.docz.site/docs/project-configuration#public) said:
![image](https://user-images.githubusercontent.com/33095380/59585271-abd7db80-9112-11e9-90c5-eb7fc836cfc3.png)

**Comment by karlwills** [comment]
Just wondering if anyone has managed to fix this? After upgrading to `1.2.0` I seem to be having the same issue:

`Module not found: Can't resolve './/public/accessibility/focus-state.png'` 

with `public: './static',` being set in doczrc.js
